### ISSUE TITLE
jobsGeneral.labels not specified in template (bug)

### ISSUE THREAD

**Issue by abix5** [nixys/nxs-universal-chart#69]
We missed it in the template. that's why this part doesn't work.

https://github.com/nixys/nxs-universal-chart/blob/main/templates/job.yml

```
apiVersion: batch/v1
kind: Job
metadata:
  name: {{ $jobName }}
  namespace: {{ $.Release.Namespace | quote }}
  labels:
    {{- include "helpers.app.labels" $ | nindent 4 }}
    # missed general.labels
    {{- with .labels }}{{- include "helpers.tplvalues.render" (dict "value" . "context" $) | nindent 4 }}{{- end }}
```

**Comment by artemdanielyan** [comment]
Hello! Thank you for noting this. I will add missing general labels in the next patch in the upcoming week.

**Comment by artemdanielyan** [comment]
Hello, fixed in 2.7.0
